### DISCUSSION TITLE
No access to my squeezelite after autoexec1 change

### DISCUSSION THREAD

**Discussion by juhanjuku** [sle118/squeezelite-esp32#405] [General]
I have successfully used squeezelite-esp32 on ESP32-A1S for couple years. 
Today I tried to access my board from Home Assistant SlimProto, but it was not discovering my player.
I found from Reddit https://www.reddit.com/r/homeassistant/comments/yypr44/anyone_using_slimproto_in_home_assistant/
suggestion to add my Home Assistant IP address at the end of AUTOEXEC1 section, what I did.
After restart I can't access my player anymore. It's IP is not answering, it's not exposing hotspot, LMS server does not see this player anymore.
Is there any way to reset my player to default settings so I can connect it again?

**Comment by ChrisCode3000** [comment]
I have a similar problem. I installed squeezelite on an esp32 a1s using the web installer. I'm using Music Assistant AddOn with the Squeezebox/Slimproto Provider inside Home Assistant as my Server. (Home Assistant OS is running on Proxmox Server)
It discovered my Logitech Squeezebox with no problems.
The ESP32 Log shows it discovered the server but it is not shown inside Music Assistant nor the Squeezer App I use on my phone.
Any suggestions what could be the issue here?
